relayer-envio-inbox/.env.example

```
@@ -0,0 +1,2 @@
+# To create or update a token visit https://envio.dev/app/api-tokens
+ENVIO_API_TOKEN="<YOUR-API-TOKEN>"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)